drivers/soft_spi/include/soft_spi_params.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SOFT_SPI_CONF_H
+#define SOFT_SPI_CONF_H
+
+#include "soft_spi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+soft_spi_t soft_spi_config[] = {
+  #ifdef SOFT_SPI_PARAMS_USER
```

Comment:
I think user params should be handled like this [1]

[1] https://github.com/RIOT-OS/RIOT/wiki/Guide:-Writing-a-device-driver-in-RIOT#default-device-configuration